tests/network_test.cpp

```diff
@@ -1261,7 +1261,6 @@ TEST_F(NetworkTest, node_full_sync) {
               nodes[0]->getDagManager()->getNumVerticesInDag().first);
     EXPECT_EQ(nodes[i]->getDagManager()->getNumVerticesInDag().first, nodes[i]->getDB()->getNumDagBlocks());
     EXPECT_EQ(nodes[i]->getDagManager()->getNumEdgesInDag().first, nodes[0]->getDagManager()->getNumEdgesInDag().first);
-    EXPECT_TRUE(!nodes[i]->getNetwork()->pbft_syncing());
```

Comment:
why this was removed?

Reply:
@mfrankovi Could you please answer @MatusKysel question? Why remove this line? Thanks!

Reply:
Some of our tests counted on the fact that once there is no new transactions being sent to the nodes there is no possibility for node to add new pbft blocks. With this change it is possible even when everything is synced up that pbft progress and that makes it possible that node can get into pbft syncing state even with no new transactions.